Jenkinsfile

```diff
@@ -33,7 +33,7 @@ pipeline {
     booleanParam(name: 'test_ci', defaultValue: false, description: 'Enable test')
     booleanParam(name: 'release_ci', defaultValue: true, description: 'Enable build the release packages')
     booleanParam(name: 'its_ci', defaultValue: true, description: 'Enable async ITs')
-    choice(name: 'build_type', choices: ['snapshot', 'staging'], description: 'Choose Snapshot or Staging Build type(Default: Snapshot)')
+    choice(name: 'build_type', choices: ['both', 'snapshot', 'staging'], description: 'Choose Snapshot or Staging Build type(Default: Both)')
```

Comment:
nit, 'both' option is not used as expression { params.build_type != 'xxx' } and this trick will work because 2 options are implemented 'snapshot' and 'staging'. Adding additional option will require more logic to be implemented.

Reply:
Yes, there are only 2 types of dra builds though. Snapshot & Staging.
So i either want both,one of them or none(and then we don't run that stage).